utils/service_manager/service.go

```diff
@@ -254,7 +254,7 @@ func StopLaunchdService(serviceName string) error {
 	return nil
 }
 
-func StopSystemServices(services []string) error {
```

Comment:
this name change isn't required, this function stops a system service based on the OS, `StopSystemServices` makes more sense